tools/frb_internal/lib/src/makefile_dart/misc.dart

```diff
@@ -111,3 +111,8 @@ String convertConfigPackage(String raw) {
   if (raw != ans) print('convertConfigPackage: $raw -> $ans');
   return ans;
 }
+
+bool isFeature(String package) {
```

Comment:
nit: maybe we can give it a more descriptive name

nit2: what about `String getRustFeaturesOfPackage(String package)` and we return the features that is needed